### PR TITLE
feat: add goku on movie

### PR DESCRIPTION
**Pull request by jemieeffendy**

### CODE REVIEW

File: dictionaryfunction.py [hunk 1]
```diff
@@ -46,7 +46,8 @@
     "Guardians of the Galaxy": 2014,
     "Avengers: Endgame": 2019, 
     "Avengers: Infinity War": 2018, 
-    "Captain Marvel": 2019
+    "Captain Marvel": 2019, 
+    "Son Goku": 2019
```
Comment (d4nstroyer) [inline dictionaryfunction.py]: Please put son goku in the right place. He is an actor.

Comment (jemieeffendy) [reply dictionaryfunction.py]: I will make a change. What is Goku's date of birth?

Comment (d4nstroyer) [reply dictionaryfunction.py]: 12 October 737

**Review by d4nstroyer — APPROVED** [review]
good work

**Pull request merged**